### PR TITLE
feat: a gqlgen & federation example

### PR DESCRIPTION
**Pull request by obmarg**

### CODE REVIEW

File: examples/gqlgen-federation/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO
```
Comment (obmarg) [inline examples/gqlgen-federation/readme.md:1]: Need to do this

File: examples/gqlgen-federation/gateway/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@grafbase/sdk": "~0.20.0"
```
Comment (fbjork) [inline examples/gqlgen-federation/gateway/package.json:13]: add grafbase as a dependency here and start/dev scripts?

Comment (fbjork) [reply examples/gqlgen-federation/gateway/package.json:13]: another idea is to use the grafbase federation gateway and pass the federated schema SDL via path?

Comment (obmarg) [reply examples/gqlgen-federation/gateway/package.json:13]: Yeah, could do.  Though I was just using `npx grafbase@latest` in `start.sh`, which seemed less annoying (having to update versions etc.)

Comment (obmarg) [reply examples/gqlgen-federation/gateway/package.json:13]: > another idea is to use the grafbase federation gateway and pass the federated schema SDL via path?

Maybe, but then I'd have to hardcode/script/instruct people how to build the federated schema SDL.  I can do that if you think we should though

Comment (fbjork) [reply examples/gqlgen-federation/gateway/package.json:13]: True, better to keep the dev server then.

**Review by fbjork — APPROVED** [review]

**Pull request merged**